### DISCUSSION TITLE
Issue with using WezTerm in combination with Tmux

### DISCUSSION THREAD

**Discussion by lominoss-git** [wezterm/wezterm#4202] [Q&A]
<img width="500" alt="wezterm-gui_2Tegk2mdvW" src="https://github.com/wez/wezterm/assets/79030093/b6295bfa-b26f-4050-9179-2ed5aacf7b0f">

Hi! As you can see, when I enter a Tmux session, WezTerm automatically inserts this string of characters. I am looking for a way to suppress this insertion. Thanks in advance.

**Comment by wez** [comment]
See https://github.com/wez/wezterm/discussions/4139
